lib/comments.go

```diff
@@ -56,7 +56,7 @@ func CompareComments(config cfg.Config, ghIssue github.Issue, jIssue jira.Issue,
 			}
 			// matches[0] is the whole string, matches[1] is the ID
 			matches := jCommentIDRegex.FindStringSubmatch(jComment.Body)
-			id, _ := strconv.Atoi(matches[1])
+			id, _ := strconv.ParseInt(matches[1], 10, 64)
```

Comment:
Yep. My upcoming PRs add features as well as more thorough error handling and I'm currently working on unit tests. It might prove difficult to slice the upcoming work into small PRs (I'll do my best obviously) but included in those changes are some minor improvements in various parts of the code.